h/api/views.py

```diff
@@ -177,11 +177,13 @@ def create(request):
                                                          legacy_appstruct)
 
     if request.feature('postgres'):
-        _publish_annotation_event(request, annotation, 'create')
-        return AnnotationJSONPresenter(request, annotation).asdict()
+        presented = AnnotationJSONPresenter(request, annotation).asdict()
+        _publish_annotation_event(request, annotation, 'create', serialized=presented)
+        return presented
```

Comment:
I think the variable name `presented`, the param name `serialized` and the method name `asdict()` are all different terms for the same thing. Maybe rename both `presented` and `serialized` to `annotation_dict`?